quarkus:dev fails on empty project

Creates an empty project, without any sources or resources and run:

```
mvn compile quarkus:dev
...
[ERROR] Failed to execute goal io.quarkus:quarkus-maven-plugin:0.13.1:dev (default-cli) on project async-demo: The project has no output yet, run `mvn compile quarkus:dev`. -> [Help 1]
```

Either the error messages must be updated to mention that the project must at least contain 1 file, or it should run and waits until we have content.

--- a/devtools/maven/src/main/java/io/quarkus/maven/DevMojo.java
+++ b/devtools/maven/src/main/java/io/quarkus/maven/DevMojo.java
@@ -186,7 +186,8 @@ public class DevMojo extends AbstractMojo {
         }
 
         if (!buildDir.isDirectory() || !new File(buildDir, "classes").isDirectory()) {
-            throw new MojoFailureException("The project has no output yet, run `mvn compile quarkus:dev`.");
+            throw new MojoFailureException("The project " + project.getName()
+                    + " has no output yet. Make sure it contains at least one source or resource file and then run `mvn compile quarkus:dev`.");
         }
 
         try {

